Automattic/wp-calypso
# Enable jetpackedgephpold and jetpackedgenew

## Proposed Changes

Update the encrypted secrets file with the new hostnames and re-enable them in the appropriate scripts.

This basically undoes #92327.

## Why are these changes being made?


Sites have been renamed, time to re-enable.

## Testing Instructions



* Decrypt the new secrets, then try running tests against `php-old` and `php-new`, e.g. `yarn workspace wp-e2e-tests build && VIEWPORT_NAME="desktop" ATOMIC_VARIATION="php-old" TEST_ON_ATOMIC="true" JETPACK_TARGET="wpcom-deployment" yarn workspace wp-e2e-tests test specs/some/test.ts`.
* Merge and observe for the scripts.

## Pre-merge Checklist
